package.json

```diff
@@ -94,6 +94,7 @@
     "moment": "^2.29.4",
     "os-locale": "^6.0.2",
     "stream-json": "^1.7.5",
+    "svelte-resize-observer": "^2.0.0",
```

Comment:
I'm not sure why we would need that dependency ?

ResizeObserver is in the API

Reply:
Good observation! I didn't see that it's actually in the API.  I'll update the PR.

Reply:
I think it's because there is a polyfill but we don't need it